contract/src/test-integration/java/io/neow3j/contract/ContractManagementIntegrationTest.java

```diff
@@ -0,0 +1,95 @@
+package io.neow3j.contract;
+
+import static io.neow3j.NeoTestContainer.getNodeUrl;
+import static io.neow3j.contract.IntegrationTestHelper.committee;
+import static io.neow3j.contract.IntegrationTestHelper.committeeWallet;
+import static io.neow3j.contract.SmartContract.getContractHash;
+import static io.neow3j.protocol.ObjectMapperFactory.getObjectMapper;
+import static io.neow3j.transaction.Signer.calledByEntry;
+import static io.neow3j.utils.Await.waitUntilTransactionIsExecuted;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import io.neow3j.NeoTestContainer;
+import io.neow3j.crypto.Base64;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.ContractManifest;
+import io.neow3j.protocol.core.methods.response.NeoGetContractState;
+import io.neow3j.protocol.http.HttpService;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.io.File;
+import java.math.BigInteger;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ContractManagementIntegrationTest {
+
+    private static Neow3j neow3j;
+    private static ContractManagement contractManagement;
+
+    private final static Path TESTCONTRACT_NEF_FILE = Paths.get("/contracts", "TestContract.nef");
+    private final static Path TESTCONTRACT_MANIFEST_FILE =
+            Paths.get("/contracts", "TestContract.manifest.json");
+
+    @ClassRule
+    public static NeoTestContainer neoTestContainer =
+            new NeoTestContainer("/node-config/config.json");
```

Comment:
In all the test classes it is no necessary to specify the path to the config.json if it has the default name, as it has here. The default constructor already uses this path.